dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ProtocolConfig.java

```diff
@@ -136,6 +136,9 @@
     // if it's default
     private Boolean isDefault;
 
+    // whether enable telnet or not
+    private Boolean enableTelnet;

```

Comment:
personally think 'boolean telnet' is enough.
`public static final String TELNET_ENABLE = "enable.telnet";` change to `public static final String TELNET_ENABLE = "telnet";`

Reply:
Personally think that it will not cause ambiguity

Reply:
There is already a property name telnet in ProtocolConfig class

Reply:
Oh, that's my fault.